dist/index.js

```diff
@@ -1 +1 @@
oup/RadioButtonGroup.jsx"))}()},function(e,t,n){"use strict";function r(e){return e&&e.__esModule?e:{default:e}}function o(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function a(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}function i(e,t){if(!e)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return!t||"object"!=typeof t&&"function"!=typeof t?e:t}function l(e,t){if("function"!=typeof t&&null!==t)throw new TypeError("Super expression must either be null or a function, not "+typeof t);e.prototype=Object.create(t&&t.prototype,{constructor:{value:e,enumerable:!1,writable:!0,configurable:!0}}),t&&(Object.setPrototypeOf?Object.setPrototypeOf(e,t):e.__proto__=t)}Object.defineProperty(t,"__esModule",{value:!0});var u=Object.assign||function(e){for(var t=1;t<arguments.length;t++){var n=arguments[t];for(var r in n)Object.prototype.hasOwnProperty.call(n,r)&&(e[r]=n[r])}return e},c=function(){function e(e,t){for(var n=0;n<t.length;n++){var r=t[n];r.enumerable=r.enumerable||!1,r.configurable=!0,"value"in r&&(r.writable=!0),Object.defineProperty(e,r.key,r)}}return function(t,n,r){return n&&e(t.prototype,n),r&&e(t,r),t}}(),s=n(2),f=r(s),p=n(0),d=r(p),_=n(127),h=r(_),m=n(1),y={SOLID:"solid",DOTTED:"dotted",DASHED:"dashed"},v=function(e){function t(){return a(this,t),i(this,(t.__proto__||Object.getPrototypeOf(t)).apply(this,arguments))}return l(t,e),c(t,[{key:"render",value:function(){var e,t=(0,f.default)(h.default.wrapper,(e={},o(e,h.default.dotted,this.props.type===y.DOTTED),o(e,h.default.dashed,this.props.type===y.DASHED),o(e,h.default.withTopMargin,this.props.withTopMargin),e));return d.default.createElement("hr",u({},(0,m.getDataAttrs)(this.props),{className:t}))}}]),t}(d.default.Component);v.defaultProps={type:y.SOLID},v.displayName="Plasma@Rule";var b=v;t.default=b;!function(){"undefined"!=typeof __REACT_HOT_LOADER__&&(__REACT_HOT_LOADER__.register(y,"type","/Users/agrozki/workspace/plasma/src/components/Rule/Rule.jsx"),__REACT_HOT_LOADER__.register(v,"Rule","/Users/agrozki/workspace/plasma/src/components/Rule/Rule.jsx"),__REACT_HOT_LOADER__.register(b,"default","/Users/agrozki/workspace/plasma/src/components/Rule/Rule.jsx"))}()},function(e,t,n){"use strict";function r(e){return e&&e.__esModule?e:{default:e}}function o(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}function a(e,t){if(!e)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return!t||"object"!=typeof t&&"function"!=typeof t?e:t}function i(e,t){if("function"!=typeof t&&null!==t)throw new TypeError("Super expression must either be null or a function, not "+typeof t);e.prototype=Object.create(t&&t.prototype,{constructor:{value:e,enumerable:!1,writable:!0,configurable:!0}}),t&&(Object.setPrototypeOf?Object.setPrototypeOf(e,t):e.__proto__=t)}Object.defineProperty(t,"__esModule",{value:!0});
```

Comment:
`/src/components/Rule/Rule.jsx"),__REACT_HOT_LOADER__.register(v,"Rule","/Users/agrozki/workspace/plasm`

Wait, what? 😱 
This seems weird, we're suppose to have hot-loader paths here?

Reply:
good catch